requirements-dev.txt

```diff
@@ -13,3 +13,4 @@ pytest-cov>=2.12.0
 pytest-env>=0.6.2
 pytest-mock>=3.6.1
 typed-ast>=1.4.3
+pyspark==3.0.1
```

Comment:
does it need to be a hard pin or will this work with `>=` ?

Reply:
I thought we had pyspark deps in setup.py?